src/cmdlib.sh

```diff
@@ -61,10 +61,14 @@ prepare_build() {
 
     echo "Using manifest: ${manifest}"
 
+    manifest_tmp_json=${workdir}/tmp/manifest.json
+    rpm-ostree compose tree --repo=repo --print-only ${manifest} > ${manifest_tmp_json}
+
     # Abuse the rojig/name as the name of the VM images
-    export name=$(manifest_get '["rojig"]["name"]')
+    export name=$(jq -r '.rojig.name' < ${manifest_tmp_json})
     # TODO - allow this to be unset
-    export ref=$(manifest_get '["ref"]')
+    export ref=$(jq -r '.ref' < ${manifest_tmp_json})
+    rm -f ${manifest_tmp_json}
```

Comment:
Any reason why we don't quote variables (here and everywhere else too)?

Reply:
Yeah, consistency is not strong on this.  Too bad there's not a "strict mode" for this.  But probably our best bet really is to rewrite this in not-bash...

Reply:
I definitely do think about quoting whenever a variable "could reasonably" have spaces; definitely when handling *externally provided* filename arguments for example.

But in practice we don't support spaces for these variables like `workdir`, `ref`, `name` etc.  So my inclination is to go with this as is - that OK with you?

Reply:
Sure, I was mostly asking for my own knowledge in case I was missing some other details, not a blocker. It makes sense to bucket this under https://github.com/coreos/coreos-assembler/issues/161.

On other coreos projects we use https://github.com/koalaman/shellcheck, which helps catching simple things.

Reply:
Ooh, wow.  That is a useful project.  I just ran it against `cmdlib.sh` and indeed...lots to fix.